extensions/wikia/Email/templates/passwordResetLink.mustache

```diff
@@ -0,0 +1,15 @@
+{{> greeting}}
+<table>
+	<tr>
+		<td class="widest">
+			<p>{{passwordIntro}}</p>
+			{{# username}}
+				<p>{{username}}</p>
+			{{/ username}}
+			<a href="{{resetLink}}">{{resetLink}}</a>
```

Comment:
Just a thought, if the link happens to be long/ugly (for example after adding the return url at the end) we may think about adding a nicely-looking version for the anchor text.